woof-distro/x86_64/ubuntu/jammy64/DISTRO_PKGS_SPECS-ubuntu-jammy

```diff
@@ -678,7 +681,7 @@ no|xsoldier|xsoldier|exe,dev>null,doc,nls
 yes|xtrans|xtrans-dev|exe>dev,dev,doc,nls||deps:yes
 no|xvidcore|libxvidcore4,libxvidcore-dev|exe,dev,doc,nls
 yes|xz|xz-utils,liblzma5,liblzma-dev|exe,dev,doc,nls||deps:yes
-no|yad||exe
+yes|yad||exe
```

Comment:
This conflicts with the yad petbuild

Reply:
Sorry, forgot that I had added those. Will remove them now.